src/ui.ts

```diff
@@ -48,6 +68,9 @@ const searchInput = document.querySelector("#search") as HTMLInputElement;
 const searchImages = () => {
     const inputValue = searchInput?.value.toLowerCase();
     const images = document.querySelectorAll("img");
+    const loadState = document.getElementById("loadState");
+    const hideLoadStateClasses = ["hidden", "w-0", "h-0"];
+    loadState?.classList.add(...hideLoadStateClasses);
```

Comment:
This is where the logic to both show and hide should be, like:

```suggestion

    const loadStateElement = document.querySelector("#loadState");
    const hideLoadStateClasses = ["hidden", "w-0", "h-0"];
    inputValue !== ""
        ? loadStateElement?.classList.add(...hideLoadStateClasses)
        : loadStateElement?.classList.remove(...hideLoadStateClasses);
```